app/views/users/following.html.erb

```diff
@@ -0,0 +1,27 @@
+   <ul class="nav nav-pills nav-justified">
+      <li class="nav-item">
+       <%= link_to "Posts", user_path(@user.username), class: "nav-link"%>
+      </li>
+      <li class="nav-item">
+      <%= link_to "Liked photos", liked_path(@user.username), class: "nav-link"%>
+      </li>
+      <li class="nav-item">
+       <%= link_to "Feed", feed_path(@user.username), class: "nav-link"%>
+      </li>
+      <li class="nav-item">
+       <%= link_to "Followers", followers_path(@user.username), class: "nav-link"%>
+      </li>
+      <li class="nav-item">
+       <%= link_to "Following", following_path(@user.username), class: "nav-link"%>
+      </li>
+    </ul>
+  </div>
+</div>
+
+<% @user.leaders.each do |leader| %>
+  <ul class ="list-group">
+  <li class="list-group-item">
+  <%=leader.username%>
```

Comment:
Please add links (using the link_to helper method) to your followers & following lists. At this stage in the project, we want to be sure that we can navigate to each listed user's page.